packages/samples/stencil/project.json

```diff
@@ -1,3 +1,29 @@
 {
-  "name": "atomic-stencil-samples"
+  "name": "atomic-stencil-samples",
+  "namedInputs": {
+    "production": [
+      "sharedGlobals",
+      "sharedTypescriptSources",
+      "{projectRoot}/stencil.config.ts",
+      "!{projectRoot}/src/components.d.ts"
+    ]
+  },
+  "targets": {
+    "cached:build:stencil": {
+      "dependsOn": ["^build"],
+      "inputs": ["^production", "production"],
+      "outputs": ["{projectRoot}/www", "{projectRoot}/src/components.d.ts"],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "node --max_old_space_size=6144 ../../../node_modules/@stencil/core/bin/stencil build",
```

Comment:
I added the max memory size here just like we do for all other Stencil build tasks.

Eventually, I think we should stop referencing `node_modules` directly. Since this project is inside `samples/`, it needs to go one level higher than the `build:stencil` targets.